### ISSUE TITLE
Add option for additional pages not within SvelteKit

### ISSUE THREAD

**Issue by ZachSaucier** [bartholomej/svelte-sitemap#45]
I use SvelteKit for my blog but not my main site. I'd like to add some additional pages by hand if possible.

I'll make a PR for adding the support!

**Comment by ZachSaucier** [comment]
PR here #46

**Comment by emilianocalzada** [comment]
Thanks @ZachSaucier 
I also need this feature to be added

**Comment by fxxmr** [comment]
I also need this feature since some of my pages are SSR and I still would like to add them to the sitemap. Thanks @ZachSaucier for the PR.
@bartholomej would it be possible to merge  #46 ?

**Comment by bartholomej** [comment]
Hi guys, thanks for your contribution. Especially Zach, of course!
I just released slightly modified version of Zachs's PR which you can see here: #51 

You can try like this: `npm install svelte-sitemap@beta` (make sure it is 2.7+ version).

if you confirm I will release stable version ;)

**Comment by pixelreseller** [comment]
Hi! Looking for this. You lib is great but i need to add some extra entries.
Tried it, like:
`npx svelte-sitemap --domain https://www.mydomain.fr --additional ['justatest','justasecondtest']`
but i don't get the expected result:
```
<url>
  <loc>https://www.albadecouverte.fr/[justatest,justasecondtest]</loc>
</url>
```

**Comment by bartholomej** [comment]
Just released `v2.7.0` with this feature request.

@pixelreseller
You seem to be calling it wrong. It's not like a *array*, but it's like this:
`npx svelte-sitemap --domain https://www.mydomain.fr -a justatest -a justasecondtest`

See this [example](https://github.com/bartholomej/svelte-sitemap?tab=readme-ov-file#%EF%B8%8F-options) in the option table.

Let me know if it works.
